tests/integration/list/resources/test_resources_command.py

```diff
@@ -16,7 +15,7 @@
 
 
 @skipIf(SKIP_STACK_OUTPUTS_TESTS, "Skip stack-outputs tests in CI/CD only")
-class TestResources(ResourcesIntegBase):
+class TestResources(DeployIntegBase, ResourcesIntegBase):
```

Comment:
Can we do the same looping behaviour with this test too please

Reply:
Changed to loop